lib/utils/__tests__/isHexColor.test.mjs

```diff
@@ -0,0 +1,13 @@
+import isHexColor from '../isHexColor.js';
+import valueParser from 'postcss-value-parser';
```

Comment:
question

Do you really need `valueParser` to test `isHexColor`?
i.e. just use the output directly

Reply:
@ybiquitous maybe we should change the type of the parameter in the JSDoc comment?
e.g. `@param {{ type: string, value: string }}`

Reply:
Yeah, you are right.
Consistency is more important.

Reply:
Not only for consistency, I think clarifying the argument type is a node of valueParser is also important. If valueParser's type changed, these tests could tell us about the change.